src/getting-started/toolchain.md

```diff
@@ -1,4 +1,10 @@
-## Rust Toolchain
+## Toolchain Installation
+
+<!---
+TODO: This needs some rework based on https://github.com/esp-rs/espup/issues/498
+
+In general in previous chapters we need to explain RISC-V vs Xtensa etc.
```

Comment:
This is explained in https://github.com/esp-rs/book/pull/200/files#diff-de5d1487c471407b3f421f00c325c14e3fe6f501b1a4b12ecad8b968276db3b9